Source/Fuse.Reactive/ObserverMap.uno

```diff
@@ -0,0 +1,182 @@
+using Uno;
+using Uno.Collections;
+
+namespace Fuse.Reactive
+{
+	//UNO: Cast to IArray was failing if this class is declared inside ObserverMap
```

Comment:
Do we have a ticket for this?

Reply:
Do now https://github.com/fusetools/uno/issues/1321
Added ref in code.